presto-cache/src/main/java/com/facebook/presto/cache/CachingFileSystem.java

```diff
@@ -147,17 +137,13 @@ public void setVerifyChecksum(boolean verifyChecksum)
     public FSDataInputStream open(Path path, int bufferSize)
             throws IOException
     {
-        return new CachingInputStream(dataTier.open(path, bufferSize), cacheManager, path, cacheValidationEnabled);
+        return dataTier.open(path, bufferSize);
     }
 
     @Override
     public FSDataInputStream openFile(Path path, HiveFileContext hiveFileContext)
```

Comment:
Do we wanna make it abstract to force all impls to think about hiveFileContext.isCacheable?

Reply:
Sounds good to me.